armi/physics/neutronics/energyGroups.py

```diff
@@ -275,14 +275,391 @@ def getGroupStructureType(neutronEnergyBoundsInEv):
 )
 
 # fmt: on
+def _create_multigroup_structures_on_finegroup_energies(
```

Comment:
interesting that we have camel_case functions in here. Anyway you're being consistent so we'll allow it!